src/backend/context.rs

```diff
@@ -821,23 +673,24 @@ impl Context {
 			None => return true,
 		};
 
-		let mut event_handlers = std::mem::replace(&mut self.windows[window_index].event_handlers, Vec::new());
+		let mut event_handlers = std::mem::take(&mut self.windows[window_index].event_handlers);
 
 		let mut stop_propagation = false;
 		event_handlers.retain_mut(|handler| {
 			if stop_propagation {
 				false
 			} else {
 				let context_handle = ContextHandle::new(self, event_loop);
-				let mut window_handle = WindowHandle::new(context_handle, event.window_id());
+
```

Comment:
Mistake I assume.

Reply:
The extra newline? Yeah, that shouldn't be there.

Reply:
Removed it :)